### DISCUSSION TITLE
petite-vue is not detected by the Vue.js devtools

### DISCUSSION THREAD

**Discussion by chienwen** [vuejs/petite-vue#215] [Q&A]
I tried to use the petite-vue on my single page with the short (dev) CDN URL. 

```html
<script src="https://unpkg.com/petite-vue"></script>
```

It works perfectly for me. But when I added more codes and tried to check the `data` by the Chrome extension tool [Vue.js devtools](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd), it shows "Vue.js not detected". 

Is this expected? If not, could someone please help me with how can I configure it to be enabled? Thank you in advance.

**Comment by halivert** [comment]
Hey @chienwen, I'd like to help you, but my dev tools are not detecting petite-vue. Can you provide a link where you give a minimal reproduction setup?

**Reply by chienwen** [reply]
Hello @halivert .

Thank you for your help. Yes, this is the code snippet I did: https://jsfiddle.net/x1s902cd

When I run it in a separate tab, i.e. https://jsfiddle.net/x1s902cd/show and try to turn on the Vue.js devtools, it shows the "Vue.js not detected" error.

**Reply by halivert** [reply]
Ok @chienwen, so currently Vue dev tools aren't able to detect petite-vue applications 😕
